python/pyspark/ml/regression.py

```diff
@@ -708,7 +710,7 @@ def _create_model(self, java_model):
 
 
 @inherit_doc
-class DecisionTreeModel(JavaModel):
+class DecisionTreeModel(JavaModel, JavaPredictionModel):
```

Comment:
@holdenk it doesn't look like anyone's doing the regression side, so should we just roll those into this one PR?